src/app/shared/components/test-certificate/test-certificate.component.ts

```diff
@@ -1,15 +1,47 @@
-import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import {
+  ChangeDetectionStrategy, Component, inject, Input, OnDestroy, OnInit,
+} from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { isTestTypeOldIvaOrMsva, toEditOrNotToEdit } from '@store/test-records';
+import {
+  Subject, takeUntil, combineLatest, filter,
+} from 'rxjs';
+import { State } from '@store/index';
+import { resultOfTestEnum } from '@models/test-types/test-type.model';
+import { TEST_TYPES_GROUP1_SPEC_TEST, TEST_TYPES_GROUP5_SPEC_TEST } from '@forms/models/testTypeId.enum';
+import { FeatureToggleService } from '@services/feature-toggle-service/feature-toggle-service';
 
 @Component({
   selector: 'app-test-certificate[testNumber][vin]',
   templateUrl: './test-certificate.component.html',
   styleUrls: ['./test-certificate.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class TestCertificateComponent {
+export class TestCertificateComponent implements OnInit, OnDestroy {
+  store: Store<State> = inject(Store<State>);
+  featureToggleService = inject(FeatureToggleService);
   @Input() testNumber!: string;
   @Input() vin!: string;
   @Input() isClickable = true;
+  certNotNeeded: boolean = false;
+  private destroyed$ = new Subject<void>();
+
+  ngOnInit(): void {
+    const isRequiredStandardsEnabled = this.featureToggleService.isFeatureEnabled('requiredStandards');
+    combineLatest([
+      this.store.pipe(select(toEditOrNotToEdit)),
+      this.store.pipe(select(isTestTypeOldIvaOrMsva)),
+    ]).pipe(
+      filter(([testResult]) => !!testResult),
```

Comment:
Below on line 38, we are checking for the existence of two values i.e. `testResult` & `isRequiredStandardsEnabled`

Is there any reason we couldn't add another clause to this filter for the feature flag? 

e.g.  `filter(([testResult]) => !!testResult && isRequiredStandardsEnabled),`

And then we should safely be able to remove the if check below? 

Happy to be told no haha, but we are doing truthy checks in two places atm so just curious